metricbeat/module/system/diskio/diskstat_windows_helper.go

```diff
@@ -224,5 +228,36 @@ func isValidLogicalDrive(path string) bool {
 	if ret == 1 || ret == 5 || err != errorSuccess {
```

Comment:
> The code filters volumes by disk type at the moment but ramdisk will show up as a fixed disk and will not be filtered out. 

So, `GetDriveTypeW` returns 3 (`DRIVE_FIXED`) for ram disks? In theory it can also return 6 (`DRIVE_RAMDISK`) :thinking:

Reply:
@jsoriano, I do see there is a separate type for it,  6 (https://docs.microsoft.com/en-us/windows/win32/api/fileapi/nf-fileapi-getdrivetypew) based on the api call but it will return type 3, even when retrieving the drive type using the cmd line (ex. wmic logicaldisk get caption,description,drivetype,providername,volumename).
I can add the type 6  in the condition but it will not help in this specific case.

Reply:
No need to add it if checking the volume label is more reliable. Thanks for the explanation.

Reply:
added it either way, along with unknown type